apiserver/deploy/base/apiserver_secure.yaml

```diff
@@ -0,0 +1,170 @@
+apiVersion: apps/v1
```

Comment:
This should be a separate "kustomization". Meaning that we should have these files copied all over. This [article](https://www.digitalocean.com/community/tutorials/how-to-manage-your-kubernetes-configurations-with-kustomize) seems to point towards a way to handle this.

Reply:
fixed